apps/dashboard/test/system/products_usr_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'application_system_test_case'
+
+class ProductsUsrTest < ApplicationSystemTestCase
+  # Only testing show and index actions for type :usr
+  # ProductsDevTest used for testing page interactions
+
+  def setup
+    stub_usr_router
+    setup_usr_fixtures
+  end
+
+  def teardown
+    teardown_usr_fixtures
+  end
+
+  test 'Index of my_shared_app can be accessed' do
+    UsrRouter.stubs(:base_path).returns(Pathname.new("test/fixtures/usr/me"))
```

Comment:
Can this go into `stub_usr_router`?  Seems like we'd want that method to give us the entire coverage.

Reply:
The problem is some other tests use that helper, and when I tried to put that line in the `stub_usr_router` helper it broke those different tests that were in some other file.